website/docs/models/index.md

```diff
@@ -154,10 +187,31 @@ nlp.enable_pipe("senter")
 The `senter` component is ~10&times; faster than the parser and more accurate
 than the rule-based `sentencizer`.
 
+#### Switch from trainable lemmatizer to default lemmatizer
+
+A number of v3.3 pipelines use a new trainable lemmatizer. You can check whether
```

Comment:
I would rephrase slightly to make this more future-proof:
```suggestion
Since v3.3, a number of pipelines use a trainable lemmatizer. You can check whether
```